worker/backend/backend.go

```diff
@@ -148,7 +216,20 @@ func (b *Backend) BulkMetrics(handles []string) (metrics map[string]garden.Conta
 //
 // Errors:
 // * Container not found.
-func (b *Backend) Lookup(handle string) (container garden.Container, err error) { return }
+func (b *Backend) Lookup(handle string) (garden.Container, error) {
+	ctx := namespaces.WithNamespace(context.Background(), b.namespace)
+
+	if handle == "" {
+		return nil, InputValidationError{}
+	}
+
+	_, err := b.client.GetContainer(ctx, handle)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Container{}, nil
```

Comment:
Missed the context on this, why do we return an empty container in this case ?

Reply:
good catch! Lookup isn't finished being implemented yet, because the Garden interface expects a `garden.Container` as a return value, but we haven't built out `backend/container.go` yet which means that we don't have anything that looks like a Garden container but does containerd stuff under the hood yet. We realized that too late into the process of writing this method though, so we just left it as is, since we wrote the unit tests around error handling already. We'll make a note to revisit this method when our container mullet is finished.

Reply:
thx